packages/style-utilities/src/styles/getFocusStyle.ts

```diff
@@ -72,7 +72,7 @@ function _getFocusStyleInternal(theme: ITheme, options: IGetFocusStylesOptions =
     borderColor = theme.palette.white,
     outlineColor = theme.palette.neutralSecondary,
     isFocusedOnly = true,
-    pointerEvents,
+    pointerEvents = 'none',
```

Comment:
It would be great to scope this fix to only DetailsList. I tried repro'ing the issue on a few controls including Button, Combobox and Dropdown, and it did not repro for any of those. The change is _probably_ harmless on other controls, but to reduce the risk of regression, it would be safer to only make the change on controls that we test.

Instead of setting a default for `pointerEvents` in `getFocusStyle`, you can add `pointerEvents: 'none'` to the options param only in DetailsList code: 
https://github.com/search?q=repo%3Amicrosoft%2Ffluentui+path%3A%2F%5Epackages%5C%2Freact%5C%2Fsrc%5C%2Fcomponents%5C%2FDetailsList%5C%2F%2F+getfocusstyle&type=code